Views/Widget-Link.liquid

```diff
@@ -1,8 +1,11 @@
-{% assign linkUrl = Model.ContentItem.Content.Link.Url.Text %}
+{% assign linkUrl = Model.ContentItem.Content.LinkDestinationPart.DestinationUrl.Text %}

```

Comment:
We still need think about backwards compatibility here. If we upgrade a site that already uses a `Link` widget, this template doesn't cater for the old `Url` text field, which means the links won't render any links unless someone manually attaches the `LinkDestinationPart` and then finds all the `Link` widgets and updates them. Obviously that's not feasible so we need to cater for the old `Url` text field still being in use.

```
{% assign linkUrl = Model.ContentItem.Content.Link.Url.Text %}

{% if Model.ContentItem.Content.LinkDestinationPart.DestinationUrl.Text != blank %}
    {% assign linkUrl = Model.ContentItem.Content.LinkDestinationPart.DestinationUrl.Text %}
{% endif %}
```